ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksPage.tsx

```diff
@@ -23,23 +42,105 @@ function ProfileChecksPage() {
+            newSelection.push(selection);
+        } else {
+            newSelection = newSelection.filter((value) => value !== selection);
+        }
+        setSearchFilter({
+            ...searchFilter,
+            [filterType]: newSelection,
+        });
+    };
+
     return (
         <>
             <PageTitle title="Compliance coverage - Profile checks" />
             <CoveragesPageHeader />
             <PageSection>
                 <CoveragesToggleGroup tableView="checks" />
             </PageSection>
-            <PageSection variant="default">
-                <PageSection variant="light" component="div">
-                    <Title headingLevel="h2">Profile results</Title>
+            <PageSection variant="default" className="pf-v5-u-py-0">
+                <PageSection variant="light" className="pf-v5-u-p-0">
+                    <Toolbar>
+                        <ToolbarContent>
+                            <ToolbarGroup className="pf-v5-u-w-100">
+                                <Title headingLevel="h2" className="pf-v5-u-py-md">
+                                    Profile results
+                                </Title>
+                            </ToolbarGroup>
+                            <ToolbarGroup className="pf-v5-u-w-100">
+                                <ToolbarItem className="pf-v5-u-flex-1">
+                                    <CompoundSearchFilter
+                                        config={searchFilterConfig}
+                                        searchFilter={searchFilter}
+                                        onSearch={onSearch}
+                                    />
+                                </ToolbarItem>
+                                <ToolbarItem align={{ default: 'alignRight' }}>
+                                    <CheckStatusDropdown
+                                        searchFilter={searchFilter}
+                                        onSelect={onSelectCheckStatus}
+                                    />
+                                </ToolbarItem>
+                            </ToolbarGroup>
+                            <ToolbarGroup className="pf-v5-u-w-100">
+                                <SearchFilterChips
+                                    filterChipGroupDescriptors={[
+                                        {
+                                            displayName: 'Profile Check',
+                                            searchFilterName: 'Compliance Check Name',
+                                        },
+                                        {
+                                            displayName: 'Cluster',
+                                            searchFilterName: 'Cluster',
+                                        },
+                                        {
+                                            displayName: 'Compliance Status',
+                                            searchFilterName: 'Compliance Check Status',
```

Comment:
Could you use `compliance.coverage.constants.ts` for `searchFilterName`?

Reply:
Oh, nice. I definitely can. I'll add this as another query string. What do you think about pulling this file up so it can be used in other sections? Maybe we can organize it depending on the search category/context?